app.js

```diff
@@ -0,0 +1,8 @@
+const sort = require("./sortingUtils/sort");
+
+const arrToSort = [6, "a", 1, "asd", 32];
+console.log("Array before sorting:");
```

Comment:
Please remember to remove debug console.logs

Reply:
Removed.